BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Views/LinkedAccounts/Delete.cshtml

```diff
@@ -0,0 +1,40 @@
+﻿@model SOEN341InstagramReplica.Models.LinkedAccount
+
+@{
+    ViewBag.Title = "Delete";
+}
+
+<h2>Delete</h2>
+
+<h3>Are you sure you want to delete this?</h3>
+<div>
+    <h4>LinkedAccount</h4>
+    <hr />
+    <dl class="dl-horizontal">
+        <dt>
+            @Html.DisplayNameFor(model => model.User.First_Name)
+        </dt>
+
+        <dd>
+            @Html.DisplayFor(model => model.User.First_Name)
+        </dd>
+
+        <dt>
+            @Html.DisplayNameFor(model => model.User1.First_Name)
+        </dt>
+
+        <dd>
+            @Html.DisplayFor(model => model.User1.First_Name)
+        </dd>
+
+    </dl>
+
+    @using (Html.BeginForm()) {
+        @Html.AntiForgeryToken()
+
+        <div class="form-actions no-color">
+            <input type="submit" value="Delete" class="btn btn-default" /> |
+            @Html.ActionLink("Back to List", "Index")
```

Comment:
localhost/LinkedAccounts/Index does not exist.
Try to add the Home controller
Something like this: @Html.ActionLink("Back to List", "Index","Home")

Reply:
understood